deployments/avalanche.diamond.json

```diff
@@ -102,11 +102,14 @@
       "ERC20Proxy": "0x5741A7FfE7c39Ca175546a54985fA79211290b51",
       "Executor": "0x2dfaDAB8266483beD9Fd9A292Ce56596a2D1378D",
       "FeeCollector": "0xB0210dE78E28e2633Ca200609D9f528c13c26cD9",
-      "LiFuelFeeCollector": "0xc4f7A34b8d283f66925eF0f5CCdFC2AF3030DeaE",
+      "LiFuelFeeCollector": "0xc02FFcdD914DbA646704439c6090BAbaD521d04C",
```

Comment:
that's fine. in some networks it might have other addresses. Please do not check if contracts have the same addresses across all networks in the future. We have a few networks where the address definitely is different so we will otherwise always run into the same conversation